Makefile

```diff
@@ -17,3 +19,23 @@ htmlcov:
 
 testcov:
 	go test -v -covermode=count -coverprofile=coverage.out ./...
+
+dockrunpq:
+	docker run -it --rm --network mynetwork postgres psql -h postgre -U postgres
+
+migrate-up:
+	migrate -database $(DB_URL) -path internal/database/migrations up
```

Comment:
NIT: As number of tools in the project grows, you should probably add a step in your Makefile for some external tools initialization/installation that would contain  `go install` and/or `go get` commands. So it would set up protobuf, migrate, gci, etc. No need to handle smth big like Docker there. Alternatively you can list such requirements in Readme and instructions to set up them and initialize the project.

Reply:
got it